FrameworkFeatureConstants.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.1' Or '$(TargetFramework)' == 'netcoreapp3.1' Or '$(TargetFramework)' == 'net5.0'">
+		<DefineConstants>$(DefineConstants);INDEXOF_CHAR_COMPARISONTYPE_SUPPORTED;TIMESPAN_MULTIPLY_SUPPORTED;NULLABLE_ATTRIBUTES_SUPPORTED</DefineConstants>
```

Comment:
Shouldnt that be INDEX_OF_CHAR_COMPARISON_TYPE_SUPPORTED ?

Reply:
18a9b7151067447f3d55d05cdb881898a92e286b